src/Apps/CollectorProfile/Routes/Saves2/Components/CreateNewListModal/ArtworkItem.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { Text, Clickable, Column, Flex, SkeletonBox, Box } from "@artsy/palette"
+import { SelectArtworkItem } from "./SelectArtworkItem"
+
+interface ArtworkItemProps {
+  item: any
+  onItemPress: (item: any) => void
+}
+
+export const ArtworkItem: React.FC<ArtworkItemProps> = ({
+  item,
+  onItemPress,
+}) => {
+  const [isSelected, setIsSelected] = React.useState(false)
+
+  const onClick = () => {
+    onItemPress(item)
+    setIsSelected(!isSelected)
+  }
+
+  return (
+    <Column span={[6, 4]}>
+      <Clickable width="100%" onClick={onClick} role="option">
```

Comment:
**question**: do we plan use `role="option"` in the future?

Reply:
Not sure, forgot to delete it, will do now!